Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -619,6 +622,25 @@
                     }
                 }
             },
+            "Small": {
+                "type": "object",
+                "required": [
+                    "type",
+                    "content",
```

Comment:
```suggestion
                    "inlineContent",
```

(Or if it's meant to actually be `content`, then I think the actual implementation is using the wrong key—testing this out with Dobri's PR now)

Reply:
Oh nice catch- thank you!